examples/quickstart-sklearn-tabular/sklearnexample/task.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+from sklearn.linear_model import LogisticRegression
+
+from flwr.common import NDArrays
+from flwr_datasets import FederatedDataset
+from flwr_datasets.partitioner import IidPartitioner
+
+# This information is needed to create a correct scikit-learn model
+UNIQUE_LABELS = [0, 1, 2]
+
+
+def get_model_parameters(model: LogisticRegression) -> NDArrays:
+    """Return the parameters of a sklearn LogisticRegression model."""
+    if model.fit_intercept:
+        params = [
+            model.coef_,
+            model.intercept_,
+        ]
+    else:
+        params = [
+            model.coef_,
+        ]
+    return params
+
+
+def set_model_params(model: LogisticRegression, params: NDArrays) -> LogisticRegression:
+    """Set the parameters of a sklean LogisticRegression model."""
+    model.coef_ = params[0]
+    if model.fit_intercept:
+        model.intercept_ = params[1]
+    return model
+
+
+def set_initial_params(model: LogisticRegression, n_classes: int, n_features: int):
+    """Set initial parameters as zeros.
+
+    Required since model params are uninitialized until model.fit is called but server
+    asks for initial parameters from clients at launch. Refer to
+    sklearn.linear_model.LogisticRegression documentation for more information.
+    """
+    model.classes_ = np.array([i for i in range(n_classes)])
+
+    model.coef_ = np.zeros((n_classes, n_features))
+    if model.fit_intercept:
+        model.intercept_ = np.zeros((n_classes,))
+
+
+fds = None  # Cache FederatedDataset
+
+
+def load_data(partition_id: int, num_partitions: int):
+    """Load the data for the given partition."""
+    global fds
+    if fds is None:
+        partitioner = IidPartitioner(num_partitions=num_partitions)
+        fds = FederatedDataset(
+            dataset="hitorilabs/iris", partitioners={"train": partitioner}
```

Comment:
```suggestion
            dataset="sckit-learn/iris", partitioners={"train": partitioner}
```

Reply:
I prefer `hitorilabs/iris` since it has a proper label format (`ClassLabel` type, so it's int instead of str).

Reply:
ok. What should we do then regarding the support? include that version of iris into our tests?

Reply:
The tests need to be redesigned. For a short-term solution, it doesn't really make any difference unless anyone cares about the warning.